src/utils/restApi/schemas.ts

```diff
@@ -2,35 +2,36 @@ import { contractId } from "@marlowe.io/runtime-core";
 import { z } from "zod";
 import { env } from "~/env.mjs";
 
+export const SWAP_TAG = "-swap-";
```

Comment:
The `-` part should be done by the function constructing the tag, not a constant

Reply:
Sure!